runtime/turing/src/xcm_config.rs

```diff
@@ -180,107 +180,13 @@ pub type Barrier = DenyThenTry<
 	),
 >;
 
-/// Based on the precedent set by other projects. This will need to be changed.
-pub fn ksm_per_second() -> u128 {
-	CurrencyId::KSM.cent() * 16
-}
-
-/// Assuming ~ $0.50 TUR price.
-pub fn tur_per_second() -> u128 {
-	let tur_equivalent = convert_to_token(CurrencyId::KSM, CurrencyId::Native, ksm_per_second());
-	// Assuming KSM ~ $130.00.
-	tur_equivalent * 260
-}
-
-parameter_types! {
-	pub UnitPerSecond: (AssetId, u128) = (
```

Comment:
i know this isn't quite done yet, but i was just curious about asset registry and poking around in orml. us removing this is because we can toss all this information into the asset registry storage? https://github.com/open-web3-stack/open-runtime-module-library/blob/master/asset-registry/src/lib.rs#L91

Reply:
Correct.  This information will be stored in the orml asset registry.  ie.  feePerSec, tokenConversionRate (KSM:Native, etc), and decimals.